### PR TITLE
Add acl pass through steps

### PR DESCRIPTION
**Pull request by akirakakar**
# Type of PR

- [x] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose

- Add steps for acl passthrough for databricks
-  
## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Author pre-publish checklist:

- [x] Added test to prove my fix is effective or new feature works
- [x] No PII in logs
- [x] Made corresponding changes to the documentation

### CODE REVIEW

**Review by shervyna — CHANGES_REQUESTED** [review]
I have 2 comments:
1. I think it's probably better to move this section within or under the "How to use the sample" section. 
2. I actually had a readme about cred passthrough(but havent mentioned the ACL part) under the /databricks folder.
https://github.com/Azure-Samples/modern-data-warehouse-dataops/tree/e2e/dataset-versioning-milestone1/e2e_samples/dataset_versioning/databricks
This will be kind of overlapping with that one, so I was thinking if you can move this section into that readme, and link the details step to that readme?

**Comment by shervyna** [comment]
image
What I mean is that in this project readme, I actually had this line about Credpassthrough and ACL, and the detail steps are directing to the /databricks readme. So you might need to either put the ACL in the /databricks readme, or fix this line :D

File: e2e_samples/dataset_versioning/README.md [hunk 1]
```diff
@@ -24,11 +24,17 @@ The solution demonstrates how to achieve the requirements described above by:
 
 We refer to [LendingClub issued Loans](https://www.kaggle.com/husainsb/lendingclub-issued-loans?select=lc_loan.csv) data hosted by Kaggle.
 
+> If you want to use you may want to modify read_csv function in main.py to customize for your sample.
```
Comment (jmostella) [inline e2e_samples/dataset_versioning/README.md:27]: use what?

File: e2e_samples/dataset_versioning/databricks/README.md [hunk 2]
```diff
@@ -7,6 +7,15 @@ Credential Passthrough is a practice that is more recommended by the official do
 
 When using a High Concurrency Cluster with credential passthrough, this cluster can be shared among multiple users. The cluster will take the identity of whoever is using the cluster and access datalake. However according to the limitations mentioned in [this doc](https://docs.microsoft.com/en-us/azure/databricks/security/credential-passthrough/adls-passthrough#--limitations), looks like MLflow on high concurrency clusters will not work well. Since we will be using MLFlow in our project, we will avoid using the high concurrency cluster and just use the standard clusters. For Standard Clusters, only a single user's identity can be tied to each cluster, so every user will need to create their own cluster when using credential passthrough.
 
+## Prerequisite
```
Comment (icchan) [inline e2e_samples/dataset_versioning/databricks/README.md:10]: In the other readme files, we used Prerequisite**s**

**Review by jmostella — CHANGES_REQUESTED** [review]
Mostly looks good. Thanks! Just one small change

File: e2e_samples/dataset_versioning/databricks/README.md (hunk 2 above)
Comment (akirakakar) [reply e2e_samples/dataset_versioning/databricks/README.md]: Fixed.

File: e2e_samples/dataset_versioning/README.md (hunk 1 above)
Comment (akirakakar) [reply e2e_samples/dataset_versioning/README.md]: Custom data.

**Review by shervyna — APPROVED** [review]
Thanks for fixing! :)

**Review by icchan — APPROVED** [review]

**Pull request merged**